Created test for conference method getCandidateReviewers

rodrigorichter opened the pull request.

gallegretti committed:
Merge remote-tracking branch 'refs/remotes/adrianob/master'

rodrigorichter committed:
Created test for conference method getCandidateReviewers

rodrigorichter committed:
Created test for conference method getCandidateReviewers

adrianob commented on src/peer_review/tests/ConferenceTest.java:
```diff
@@ -37,6 +38,23 @@ public void testsortReviewers() {
 	}
+	
+	@Test
+	public void testgetCandidateReviewers() {
```
Isso deveria estar no ResearcherTest, pois de fato estamos testando o isEligibleToReview().

gallegretti committed:
Fix allocateArticle function

It should be removing the article from the submitted list and adding
into the allocated list

rodrigorichter committed:
updated test for getCandidateReviewers

gallegretti committed:
WIP AllocateArticleToMemberCommand

adrianob committed:
implement select article command

adrianob committed:
Merge pull request #23 from gallegretti/master

Fix allocateArticle function

adrianob committed:
fix conflicts

gallegretti committed:
Merge pull request #24 from adrianob/selectArticle

implement select article command

adrianob committed:
rename Grade to Review and fix duplicated articles

adrianob committed:
remove useless fields

adrianob committed:
fix tests

gallegretti committed:
Merge pull request #25 from adrianob/renameGrade

rename Grade to Review and fix duplicated articles

rodrigorichter committed:
fix

rodrigorichter committed:
Fixed test for getCandidateReviewers
